yarn.lock

```diff
@@ -4504,6 +4504,11 @@ core-js-pure@3.1.3, core-js-pure@^3.0.1:
   resolved "https://registry.yarnpkg.com/core-js-pure/-/core-js-pure-3.1.3.tgz#4c90752d5b9471f641514f3728f51c1e0783d0b5"
   integrity sha512-k3JWTrcQBKqjkjI0bkfXS0lbpWPxYuHWfMMjC1VDmzU4Q58IwSbuXSo99YO/hUHlw/EB4AlfA2PVxOGkrIq6dA==
 
+core-js-pure@^3.0.0:
```

Comment:
core-js-pure?

Reply:
That's correct. `@babel/runtime-corejs3` uses `core-js-pure`:

https://github.com/babel/babel/blob/33ab4f166117e2380de3955a0842985f578b01b8/packages/babel-runtime-corejs3/package.json#L11-L14

The old version (`@babel/runtime-corejs2`) used `core-js`:

https://github.com/babel/babel/blob/33ab4f166117e2380de3955a0842985f578b01b8/packages/babel-runtime-corejs2/package.json#L11-L14